app/javascript/routes/LeaseUpApp.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+
+import LeaseUpApplicationsPage from '~/components/lease_ups/LeaseUpApplicationsPage'
+import LeaseUpListingsPage from '~/components/lease_ups/LeaseUpListingsPage'
+import ApplicationPage from '~/components/applications/ApplicationPage'
+
+const LeaseUpApp = () => {
+  return (
+    <div>
+      <Switch>
+        <Route path='/lease-ups/listings/:listing_id'>
+          <LeaseUpApplicationsPage />
+        </Route>
+        <Route path='/lease-ups/listings'>
+          <LeaseUpListingsPage />
+        </Route>
+        <Route path='/lease-ups/applications/:application_id/supplemental'>
+          <LeaseUpListingsPage />
```

Comment:
Should this be the SupplementalApplicationPage?

also it looks like links from the applications table are still going to: https://dahlia-lap-full-pr-416.herokuapp.com/applications/a0o0P00000JlYj8QAF/supplementals, with no /lease-ups. Is that coming in a separate PR?

Actually looks like these routes don't work yet. Should I just totally ignore this part for now?